### PR TITLE
Works and About Myself features

### PR DESCRIPTION
**Pull request by ommarz**
[Add] Works feature to show personal projects.
[Add] About Myself feature to show personal skills.
[Add] Styling similar to the template.
[Add] images
[Updated] index.html classes' names to improve styling

### CODE REVIEW

**Comment by ommarz** [comment]
I still have a couple of divider elements missing. It'd be great if you could share some examples or resources on how to do it.
image

File: stylesheet.css [hunk 1]
```diff
@@ -97,3 +97,371 @@
...
+  margin-top: 10px;
+  margin-left: 20px;
+  font-family: 'Inter', sans-serif;
+  color: #fff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 30px;
+  letter-spacing: 0.03em;
+}
```
Comment (brainconnect93) [inline stylesheet.css:208]: - [x] Great effort working on your project, but I would kindly suggest you follow the Figma design strictly as much as possible. I noticed the button the first card is not properly aligned with text and I would like you to work on it and be sure it looks like what we have in the Figma Design.

- [x] Also kindly work on the list `i.e HTML, CSS, JavaScript etc`of text and make them align in the center of the border as seen in the Figma design 

Your Design :point 

Screenshot_20230610_230202

File: stylesheet.css [hunk 2]
```diff
@@ -97,3 +97,371 @@
...
+  margin-left: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
```
Comment (brainconnect93) [inline stylesheet.css:386]: - [x] I would kindly suggest you work on the button `Get resumi` and let it aligned with other text as seen in the Figma design. 
- [x] Also I would kindly advice you give spaces between the the cards like margin top and bottom
- [x] Also kindly give all the buttons some kind of effect like `:hover, :active and :enabled`

Your Design :point_down: 

Screenshot_20230610_230603

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @ommarz, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
-  Create the project cards and well aligned✔️
- Create the About me section with the tech stack ✔️
- Descriptive and professional README file ✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ommarz** [comment]
@brainconnect93 Improved all suggestions made in review and improved styling to make sure to be as close as possible to the template

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @ommarz   👋

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Good job with the UI✔️ 
- Kudos to meet the suggested changes   👍 

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

- [x] For the weblint error, you can try changing the version to `12`, and that will likely fix the issue you face 👍 

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**